tests/zkevm/test_worst_compute.py

```diff
@@ -105,7 +114,93 @@ def test_worst_keccak(
     )
 
 
-@pytest.mark.zkevm
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize(
+    "gas_limit",
+    [
+        Environment().gas_limit,
+    ],
+)
+@pytest.mark.parametrize(
+    "address,static_cost,per_word_dynamic_cost,bytes_per_unit_of_work",
+    [
+        pytest.param(0x02, 60, 12, 64, id="SHA2-256"),
+        pytest.param(0x03, 600, 120, 64, id="RIPEMD-160"),
```

Comment:
I didn't find a way to pull the static and dynamic costs from precompiles. Is there a way to do this? Mostly asking for future-proofness

Reply:
We could add it to the gas table similar [keccak](https://github.com/ethereum/execution-spec-tests/blob/328b7f49f56e0a1b05d915f4e6a918e8391d22db/src/ethereum_test_forks/forks/forks.py#L122).

Reply:
That would be quite useful yes.